src/Hl7.Fhir.Core/Model/ResourceNames.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Hl7.Fhir.Model
+{
+    internal class ResourceNames
```

Comment:
These aren't all resources - FhirTypeNames is maybe a better name?

Reply:
I agree